docs/examples/security/using_abstract_authentication_middleware/authentication_middleware.py

```diff
@@ -0,0 +1,29 @@
+from docs.examples.security.using_abstract_authentication_middleware.main import MyToken, MyUser
```

Comment:
I think this is not great for an example. Maybe could use a relative import to make it easier to read?

Reply:
I rewrote in a bottle spirit, lmk if it looks better